test/e2e/specs/appearance/widgets__legacy-visibility.ts

```diff
@@ -20,8 +20,9 @@ declare const browser: Browser;
 
 // We're skipping for Atomic sites for now because the currently used theme
 // doesn't support widgets.
-skipDescribeIf( envVariables.TEST_ON_ATOMIC )(
-	DataHelper.createSuiteTitle( 'Widgets' ),
+// Mobile viewport is skipped due to https://github.com/Automattic/wp-calypso/issues/64536.
+skipDescribeIf( envVariables.TEST_ON_ATOMIC || envVariables.VIEWPORT_NAME === 'mobile' )(
```

Comment:
I've moved the skip clause if the viewport size is mobile up to the top-level `describe` block.

It felt silly, as mobile viewport, to go through the setup for this spec only to not actually execute the all-important check.